momento/dictionary_test.go

```diff
@@ -148,6 +146,18 @@ var _ = Describe("Dictionary methods", func() {
 		Entry("both nil", nil, nil),
 	)
 
+	It("try using negative ttl for set", func() {
```

Comment:
It should read like a sentence describing the expected behavior.

```suggestion
	It("errors with a negative ttl", func() {
```